tsconfig.json

```diff
@@ -1,39 +1,34 @@
 {
   "compilerOptions": {
     "allowJs": true,
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
-    "esModuleInterop": true,
-    "inlineSources": true,
+    // ensures single-file compilers, like babel, webpack+swc, etc work
     "isolatedModules": true,
+    // makes typechecking faster (also see `tsBuildInfoFile` setting)
+    "incremental": true,
```

Comment:
This is pretty much a fancy cache (in our case).

In my local testing it speeds up successive type checking by 3-4x!